backend/tracim_backend/config.py

```diff
@@ -798,6 +824,22 @@ def _set_default_app(self, enabled_app_list: typing.List[str]) -> None:
             app_config=self,
         )
 
+        office_document = Application(
+            label="Office Document",
```

Comment:
rename to Collaborative document edition, also in slug and main route `/ui/workspaces/{workspace_id}/office` => `/ui/workspaces/{workspace_id}/online-collaborative-edition`

Reply:
behavior of this path need to be decided also.